### ISSUE TITLE
ERROR api: CSRF-token verification failed for post (error 222)

### ISSUE THREAD

**Issue by TehShrike** [keybase/client#9631]
Keybase GUI Version: 1.0.34-20171027003854+8e989533a

When attempting to use the CLI to recover my passphrase, or use the GUI to reset my passphrase, the request fails with "ERROR api post to passphrase/sign error: CSRF-token verification failed for post (error 222)"

**Comment by TehShrike** [comment]
log id 3df927f42d0e23368b8b791c

**Comment by maxtaco** [comment]
@patrickxb can you help @TehShrike get a call to `sesscheck.json`?  That will be enough to reset this issue, but I wanted to make sure he doesn't have to logout/login, since he needs to recover his passphrase.

**Comment by patrickxb** [comment]
@TehStrike restarting kbfs will force a call to sesscheck.  Easiest way would be to kill the `kbfs` process and let launchd restart it.

    killall kbfs

should do the trick.

**Comment by TehShrike** [comment]
That worked, thank you!

I'm not sure if this issue should be left open to track the underlying cause?

**Comment by maxtaco** [comment]
This issue will fade in the future, but some people might be stuck in an awkward place during the transition.  @patrickxb we might consider a one-time workaround for an upcoming release

**Comment by patrickxb** [comment]
Yeah, I think we could make `keybase status` force it to happen, for example.  All that is required is an RPC call to `CurrentSession`, which only `kbfs` does.

Everything else goes out of its way to not call `sesscheck` until the local session expires.
